### ISSUE TITLE
Port twisted.python.win32 to Python 3 (minimally)

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6039]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6039|
|Type|enhancement|
|Created|2012-10-03 11:43:09Z|
|Branch|https://github.com/twisted/twisted/tree/minimal-win32-6039|

`twisted.python.win32` is required by some existing ported modules; it should be ported to Python sufficiently to meet its non-Windows function of providing some importable constants.

<details><summary>Searchable metadata</summary>

```
trac-id__6039 6039
type__enhancement enhancement
reporter__itamar itamar
priority__normal normal
milestone__Python_3_3_Minimal Python 3.3 Minimal
branch__branches_minimal_win32_6039 branches/minimal-win32-6039
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1349264589000000 1349264589000000
changetime__1349982566000000 1349982566000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [36074]) Branching to 'minimal-win32-6039'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [36075]) Could potentially have been shorter, but three lines is pretty _close_ to minimal.

refs [#6039](https://github.com/twisted/twisted/issues/6039)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @itamarst<br>@exarkun set status to `new`|
|-|-|

Extremely small changeset.  [Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/minimal-win32-6039).

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_5"></a>|@itamarst set owner to @exarkun|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [36102]) Merge minimal-win32-6039

Author: exarkun
Reviewer: itamarst
Fixes: [#6039](https://github.com/twisted/twisted/issues/6039)

Minimum changes to `twisted.python.win32` to be able to include it in the Python 3
version of Twisted.  However, the module has not actually been ported to or tested
on Python 3.
